common/src/main/java/tachyon/network/protocol/RPCBlockReadRequest.java

```diff
@@ -74,7 +82,8 @@ public void validate() {
 
   @Override
   public String toString() {
-    return "RPCBlockReadRequest(" + mBlockId + ", " + mOffset + ", " + mLength + ")";
+    return "RPCBlockReadRequest(" + mBlockId + ", " + mOffset
```

Comment:
This would be more readable with `String.format("RPCBlockReadRequest(%s, %s, %s, %s, %s)", ...)`